rllib/algorithms/algorithm_config.py

```diff
@@ -214,6 +228,9 @@ def __init__(self, algo_class=None):
         self._disable_action_flattening = False
         self._disable_execution_plan_api = True
 
+        # Has this config object been frozen (cannot alter its attributes anymore).
```

Comment:
AlgorithmConfig objects can now be frozen (all Algorithms do this in their ctor now) to make sure no one can alter the config anymore once passed into an Algo's c'tor.